packages/desktop/package.json

```diff
@@ -55,7 +55,7 @@
     "@babel/core": "*",
     "@babel/preset-env": "*",
     "@electron/notarize": "^3.0.0",
-    "@electron/rebuild": "3.6.0",
+    "@electron/rebuild": "3.6.1",
```

Comment:
There's a fix on 3.6.1 which might affect rebuilding for different platforms https://github.com/electron/rebuild/pull/1153